packages/react-components/src/components/DetailsCard/DetailsCard.tsx

```diff
@@ -25,7 +25,7 @@ export interface IDetailsCardProps {
   /**
    * Set the label
    */
-  label: string;
+  label: string | React.ReactNode;
```

Comment:
only React.ReactNode will be enough
```suggestion
  label: React.ReactNode;
```